include/swift/AST/DiagnosticsSema.def

```diff
@@ -3000,6 +3000,14 @@ ERROR(witness_requires_class_implementation,none,
       "cannot be implemented in a protocol extension because it returns 'Self' "
       "and has associated type requirements",
       (RequirementKind, const ValueDecl *, Type))
+ERROR(implicit_init_witness_not_accessible_proto,none,
+      "Implicit memberwise initializer %1 is declared internal because it was "
+      "synthesized. This conflicts with the requirement that it be declared "
+      "%select{%error|fileprivate|internal|package|public|%error}3 because it matches a "
+      "requirement in %select{private|fileprivate|internal|package|public|%error}4 protocol "
+      "%5",
+      (RequirementKind, const ValueDecl *, bool, AccessLevel, AccessLevel,
+       const ProtocolDecl *))
```

Comment:
```suggestion
ERROR(implicit_init_witness_not_accessible_proto,none,
      "Implicit memberwise %kind1 is declared internal because it was "
      "synthesized. This conflicts with the requirement that it be declared "
      "%select{%error|fileprivate|internal|package|public|%error}3 because it matches a "
      "requirement in %select{private|fileprivate|internal|package|public|%error}4 protocol "
      "%5",
      (RequirementKind, const ConstructorDecl *, bool, AccessLevel, AccessLevel,
       const ProtocolDecl *))
```